Sources/Networking/HTTPClient/HTTPClient.swift

```diff
@@ -666,17 +668,15 @@ extension HTTPClient {
         return true
     }
 
-    internal func isRetryable(_ urlResponse: HTTPURLResponse) -> Bool {
+    internal func isResponseRetryable(_ urlResponse: HTTPURLResponse) -> Bool {
         let isStatusCodeRetryable = self.retriableStatusCodes.contains(urlResponse.httpStatusCode)
-        let isRetryableString = urlResponse.value(forHTTPHeaderField: ResponseHeader.isRetryable.rawValue)
-        let isRetryable: Bool
-        if let isRetryableString {
-            isRetryable = Bool(isRetryableString.lowercased()) ?? true
-        } else {
-            isRetryable = true
+        let doesServerAllowRetryString = urlResponse.value(forHTTPHeaderField: ResponseHeader.isRetryable.rawValue)
+        var doesServerAllowRetry: Bool = true
+        if let doesServerAllowRetryString = doesServerAllowRetryString {
+            doesServerAllowRetry = Bool(doesServerAllowRetryString.lowercased()) ?? true
```

Comment:
I'm sure this version is more clever, but it was kinda clearer before the ternary IMO

Reply:
I'm okay reverting it! I'll keep the name `doesServerAllowRetry` over `isRetryable` though 😅